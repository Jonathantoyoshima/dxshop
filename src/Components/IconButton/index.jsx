import { Button, Icon } from './styles';

const CartButton = ({position, src, to, alt}) => {

  return (
    <Button to={to} position={position}>
      <Icon src={src} alt={alt}/>
    </Button>
  );
};

export default CartButton;
