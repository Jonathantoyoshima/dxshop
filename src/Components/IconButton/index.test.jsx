import React from 'react';
// import {Router} from "react-router-dom";
import { shallow } from 'enzyme';
import IconButton from './index';
import Cart from 'Images/icon/cart.svg';

test('Render Build Icon Button Snapshot', () => {
  const component = shallow(<IconButton src={Cart} to={'/checkout'} />);
  expect(component).toMatchSnapshot();
});
