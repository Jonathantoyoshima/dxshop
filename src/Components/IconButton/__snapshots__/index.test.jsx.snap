// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Build Icon Button Snapshot 1`] = `ShallowWrapper {}`;
